dwave/cloud/api/client.py

```diff
@@ -131,9 +133,108 @@ def request(self, method: str, *args, **kwargs):
         return response
 
 
-class VersionedAPISession(LoggingSession):
+class PayloadCompressingSession(LoggingSession):
+    """A :class:`requests.Session` subclass (technically, a further specialized
+    :class:`.LoggingSession`) that adds support for payload compression on the
+    fly.
+
+    Args:
+        compress (bool, default=False):
+            Preemptively compress HTTP request body, even on PUT and POST requests.
+            When compression is enabled, the payload is always compressed in 32KiB
```

Comment:
```suggestion
            When compression is enabled, the payload is always compressed in 32KB
```

Reply:
KiB is the correct technical term here, and I'm sticking to it. :slightly_smiling_face: